alibi_detect/utils/perturbation.py

```diff
@@ -561,7 +561,8 @@ def zoom_blur(x: np.ndarray, max_zoom: float, step_zoom: float, xrange: tuple =
         return x_z
 
 
-def glass_blur(x: np.ndarray, sigma: float, max_delta: int, iterations: int, xrange: tuple = None) -> np.ndarray:
+def glass_blur(x: np.ndarray, sigma: float, max_delta: int, iterations: int,
+               channel_axis: int = -1, xrange: tuple = None) -> np.ndarray:
```

Comment:
The public interface has now changed to explicitly include `channel_axis`, although it's unclear whether the implementation would respect anything other than the default `-1` value (it assumes the layout starts with `[height, width]` axes). Still the function is not used anywhere publicly in the code base or examples so I think this is fine.

Reply:
If we're not sure `channel_axis` actually works for anything other than `-1`, would it not be safer to remove it as a kwarg and just hard-code it inside `glass_blur` and `gaussian_blur`? i.e. just say in docstrings that these functions assume `[h, w, c]`?

Reply:
Agreed, updated here: https://github.com/SeldonIO/alibi-detect/pull/857/commits/ceca05a2b28f7ddfbf0069664ec05c1f65863211